libs/membership/website/src/lib/subscribe/subscribe-container.tsx

```diff
@@ -69,14 +72,28 @@ export const SubscribeContainer = <T extends OptionalKeysOf<RegisterMutationVari
   })
 
   const [register] = useRegisterMutation({
-    onError: () => challenge.refetch(),
+    onError: async error => {
+      const emailAlreadyInUseError = error.graphQLErrors.find(
```

Comment:
Probably this should be checked globally and not in this container. We can check this via Apollo links or error handlers iirc